docs/modules/ROOT/pages/references/architecture/identity-data-model.adoc

```diff
@@ -17,14 +17,21 @@ Each {product} user is represented by a user object in Keycloak.
 Organizations::
 Each {product} organization is represented by a group object in Keycloak.
 
+Teams::
+Each team of an {product} organization is represented by a group object in Keycloak.
+A group object set below an organization group object in the group hierarchy becomes a team.
```

Comment:
In #63 we're already using the word "represented" when talking about CRD/specs. Could be misleading here.
```suggestion
Teams::
Each team of an {product} organization is implemented as a group object in Keycloak.
A group object set below an organization group object in the group hierarchy becomes a team.
```

Reply:
Thanks for the input 😊